joblib/pool.py

```diff
@@ -315,15 +316,15 @@ class CustomizablePicklingQueue(object):
 
     This class is an alternative to the multiprocessing implementation
     of SimpleQueue in order to make it possible to pass custom
-    pickling reducers, for instance to avoid memory copy when passing
+    pickling reducers. For instance, it avoids memory copy when passing
     memory mapped datastructures.
 
-    `reducers` is expected expected to be a dictionary with key/values
-    being `(type, callable)` pairs where `callable` is a function that
-    give an instance of `type` will return a tuple `(constructor,
-    tuple_of_objects)` to rebuild an instance out of the pickled
-    `tuple_of_objects` as would return a `__reduce__` method. See the
-    standard library documentation on pickling for more details.
+    `reducers` is expected to be a dictionary with key/values
+    being `(type, callable)` pairs where `type` is an instance and `callable`
+    is a function that returns a tuple `(constructor, tuple_of_objects)`
```

Comment:
I am wondering whether the original sentence was not supposed to be "that given a function of ...", i.e. the callable takes `type` as input argument and returns a tuple.